.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  push_to_registries:
+    name: Push Docker image to GitHub Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set tag in GitHub environment
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CONTAINER_REGISTRY_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Yeah, I would also go for Dockerhub.

Reply:
The reason why I investigated publishing to GitHub container registry (ghcr) in the first place is because we need to track down the credentials to the appropriate DockerHub account before being able to publish there.
But also, one doesn't forbid the other.
This action can easily be extended to publish to both ghcr and DockerHub.
I would tend to think it is better to publish in both places. Especially more so given that ghcr gives us a good interim solution while we find out about the DockerHub account.

Reply:
Using GHCR  seems like a great interim solution until you get the credentials, but publishing to both places can often be confusing for community members trying to figure out which one is the official image.